pkg/controller/tektoninstallation/tektoninstallation_controller_test.go

```diff
@@ -181,3 +320,20 @@ func newReconcileRequest(tektonInstallation *v1alpha1.TektonInstallation) reconc
 func generateName(prefix string) string {
 	return fmt.Sprintf("%s-%d", prefix, time.Now().UnixNano())
 }
+
+// NewTektonConfig returns a new TektonConfig with the given conditions
+func NewTektonConfig(conditions ...config.ConfigCondition) *config.Config {
```

Comment:
could this function take a list of ConfigCondition codes? then we could call this function with: 
```go
NewTektonConfig("applied-addons", config.InstalledStatus, "validated-pipeline")
```
it would make the tests much shorter and easier to read

Reply:
@MatousJobanek the func now takes one or more strings..